lib/sigil.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Jason.Sigil do
+  @doc ~S"""
+  Handles the sigil `~j` for JSON strings.
+
+  Calls `Jason.decode!/2` with modifiers mapped to options.
+
+  ## Modifiers
+
+    * `a` - keys are converted to atoms using String.to_atom/1
+    * `A` - keys are converted to atoms using String.to_existing_atom/1
+    * `r` - when possible tries to create a sub-binary into the original
+    * `c` - always copies the strings
```

Comment:
Let's describe those in terms of equivalent options from `Jason.decode`

Reply:
Fixed: referring to Jason.decode/2 instead now.